test/conformance/source/environment.cpp

```diff
@@ -199,6 +202,19 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get the argument (test_devices_count) to limit test devices count.
+    u_long count_set = 0;
+    if (argc >= 3) {
+        count_set = std::strtoul(argv[2], nullptr, 10);
```

Comment:
is this always the second argument?

Reply:
It depends on cts_exe.py scripts. In this case, it will always be the second argument.

Reply:
Right, but the gtest binary may be executed manually, the script can change etc. You shouldn't be hard-coding such assumptions.

Reply:
I've changed it. What about that?